src/query/storage/m3/storage.go

```diff
@@ -398,6 +400,15 @@ func (s *m3storage) SearchSeries(
 	}, nil
 }
 
+// CompleteTagsCompressed has the same behavior as CompleteTags.
```

Comment:
Is it the same return type as well?  What's the difference between `CompleteTags` and `CompleteTagsCompressed`?

Reply:
It's just to allow `m3storage` to satisfy `m3.Querier`, which the rpc server has been refactored to receive, to allow a more minimal implementation of the comparator itself.

Reply:
I see np